.github/workflows/coding-standards.yml

```diff
@@ -3,10 +3,11 @@
 name: "Check Coding Standards"
 
 on:
+  create:
   pull_request:
   push:
     branches:
-      - "master"
+      - "*.x"
```

Comment:
Let's drop any filter on `branches:` overall (can leave it empty and it should pick up everything)